microsoft-edge/toc.yml

```
@@ -567,6 +567,12 @@
                 - name: Display or edit source files using the Quick source tool
                   href: devtools-guide-chromium/quick-source/quick-source-tool.md
                   displayName: Quick source tool  # also indexed elsewhere in this file
+# Recorder tool -----------------------------------------------------------
+            - name: Recorder tool  # new jump page
```

```suggestion
            - name: Recorder tool
```
parent node is null, doesn't bring up an article/page

---

removed comment, shortened comment above it